keras/optimizers.py

```diff
@@ -333,17 +336,21 @@ def get_config(self):
 
 
 class Adadelta(Optimizer):
-    """Adadelta optimizer.
+    """Adadelta optimizer. Adadelta is a more robust extension of Adagrad
+    that adapts learning rates based on a moving window of gradient updates,
+    instead of accumulating all past gradients. Using Adadelta, you don't
+    even have to choose an initial learning rate.
```

Comment:
"don't even have to choose an initial learning rate." -> that's the dream, but whether Adadelta achieves this is debatable. Please make this statement more hypothetical

Reply:
what I meant is that `lr` doesn't appear in the original paper. so that's "achieved". of course, whether that leads to improved performance is a different story.